pkg/api/v1alpha2/types_config.go

```diff
@@ -172,17 +172,28 @@ func (o Operator) GetUniqueName() (string, error) {
 		}
 		uniqueName = path.Join(uniqueName, name)
 	}
-	if tag != "" {
-		uniqueName = uniqueName + ":" + tag
+
+	if tag == "" && id == "" {
+		// if we get here without a tag or digest, default to latest tag
+		uniqueName = uniqueName + ":" + "latest"
```

Comment:
OCPBUGS-2633 is one case among many where the use of latest by default makes oc-mirror fail.